src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionEmptyBlockTest.java

```diff
@@ -36,6 +36,8 @@ protected String getCheckName() {
         return checkName;
     }
 
+    // until https://github.com/checkstyle/checkstyle/issues/11722
+    @Deprecated(since = "10.2")
```

Comment:
As I am relooking at this, what is actually deprecated?
I am seeing `addAttribute` as the only thing deprecated and it is our code and our test. This is even xpath tests which aren't really use for code coverage or pitest. Why don't we swap this out for `addProperty`?

This might be a good example to describe what is deprecated with these annotations.

Reply:
awesome catch, fixed.